backend/dal/Services/CoordinateTransformService.cs

```diff
@@ -0,0 +1,87 @@
+using ProjNet.CoordinateSystems.Transformations;
+using System.Collections.Generic;
+using System;
+
+namespace Pims.Dal.Services
+{
+    public class CoordinateTransformService : ICoordinateTransformService
+    {
+        protected readonly CoordinateSystemFactory CoordinateSystemFactory = new CoordinateSystemFactory();
+        protected readonly CoordinateTransformationFactory CoordinateTransformationFactory = new CoordinateTransformationFactory();
+        private readonly Dictionary<int, CoordinateSystem> _projections = new Dictionary<int, CoordinateSystem>();
+
+        private static IEnumerable<KeyValuePair<int, string>> DefaultProjections()
+        {
+            yield return new KeyValuePair<int, string>(4326, @"GEOGCS[""WGS 84"",DATUM[""WGS_1984"",SPHEROID[""WGS 84"",6378137,298.257223563,AUTHORITY[""EPSG"",""7030""]],AUTHORITY[""EPSG"",""6326""]],PRIMEM[""Greenwich"",0,AUTHORITY[""EPSG"",""8901""]],UNIT[""degree"",0.0174532925199433,AUTHORITY[""EPSG"",""9122""]],AUTHORITY[""EPSG"",""4326""]]GEOGCS[""WGS 84"",DATUM[""WGS_1984"",SPHEROID[""WGS 84"",6378137,298.257223563,AUTHORITY[""EPSG"",""7030""]],AUTHORITY[""EPSG"",""6326""]],PRIMEM[""Greenwich"",0,AUTHORITY[""EPSG"",""8901""]],UNIT[""degree"",0.0174532925199433,AUTHORITY[""EPSG"",""9122""]],AUTHORITY[""EPSG"",""4326""]]");
+            yield return new KeyValuePair<int, string>(3005, @"PROJCS[""NAD83 / BC Albers"",GEOGCS[""NAD83"",DATUM[""North_American_Datum_1983"",SPHEROID[""GRS 1980"",6378137,298.257222101,AUTHORITY[""EPSG"",""7019""]],TOWGS84[0,0,0,0,0,0,0],AUTHORITY[""EPSG"",""6269""]],PRIMEM[""Greenwich"",0,AUTHORITY[""EPSG"",""8901""]],UNIT[""degree"",0.0174532925199433,AUTHORITY[""EPSG"",""9122""]],AUTHORITY[""EPSG"",""4269""]],PROJECTION[""Albers_Conic_Equal_Area""],PARAMETER[""standard_parallel_1"",50],PARAMETER[""standard_parallel_2"",58.5],PARAMETER[""latitude_of_center"",45],PARAMETER[""longitude_of_center"",-126],PARAMETER[""false_easting"",1000000],PARAMETER[""false_northing"",0],UNIT[""metre"",1,AUTHORITY[""EPSG"",""9001""]],AXIS[""Easting"",EAST],AXIS[""Northing"",NORTH],AUTHORITY[""EPSG"",""3005""]]");
+        }
+
+        private CoordinateSystem CreateCoordinateSystem(string wkt)
+        {
+            try
+            {
+                return CoordinateSystemFactory.CreateFromWkt(wkt);
+            }
+            catch (Exception)
+            {
+                // as a fallback we ignore projections not supported
+                return null;
+            }
+        }
+
+        /// <summary>
+        /// Creates an instance of this class
+        /// </summary
+        public CoordinateTransformService()
+        {
+            var enumeration = DefaultProjections();
+            foreach (var sridWkt in enumeration)
+            {
+                var cs = CreateCoordinateSystem(sridWkt.Value);
+                if (cs != null)
+                {
+                    _projections.Add(sridWkt.Key, cs);
+                }
+            }
+        }
```

Comment:
This is probably fine, but it looks like commented out code on git

Reply:
hmm - looks weird it should not be commented out.

Reply:
![image](https://user-images.githubusercontent.com/24322224/172508191-7a0057f1-953f-47d1-b1c0-b6ccfc9ffad6.png)

Reply:
Looks/works good on VS Code - perhaps github colorization gets confused by the documentation?